### DISCUSSION TITLE
Elevation Problem

### DISCUSSION THREAD

**Discussion by marcoma9023** [pelicanmapping/osgearth#1914] [Support Q&A]
Hi Glenn,

can you explane me the logic with multiple elevation layers?

Then, I have a problem....in my application I integrated Triton for the sea....

By default when I add the TritonLayer, the useHeightMap = true....

Only if I integrate the bathimetric data into the map the waves are showed....is it correct?

But the problem is that, using the bathymetric tiff, the ground is below the sea level.....and this is wrong....

https://user-images.githubusercontent.com/87537804/147123906-7ca1e2e0-54c9-4360-b5b5-62e991040265.mp4

Then, what is the correct way to put the Triton Layer preventing the sea from going to land?

Thanks

**Comment by gwaldron** [comment]
When you use the `TritonLayer` with `useHeightMap=true`, osgEarth will sample the elevation data in the map and apply transparency to the ocean geometry. An elevation of zero gives full transparency, and the transparency ramps down as the elevation decreases into the sea bed.

This is not a very accurate approach, and it doesn't work for inland water bodies, so for close-up views like yours you can instead (or in addition) apply a mask. The example earth file `triton.earth` shows how to do this.
